Common/UnitDefinitions/CoefficientOfThermalExpansion.json

```diff
@@ -0,0 +1,46 @@
+﻿{
+  "Name": "CoefficientOfThermalExpansion",
+  "BaseUnit": "InverseKelvin",
+  "XmlDoc": "A unit that represents a fractional change in size in response to a change in temperature.",
+  "BaseDimensions": {
+    "Θ": -1
+  },
+  "Units": [
+    {
+      "SingularName": "InverseKelvin",
+      "PluralName": "InverseKelvin",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "K⁻¹" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "InverseCelsius",
```

Comment:
InverseDegreeCelsius and InverseDegreesCelius?
Matches Temperature and TemperatureDelta. 

https://github.com/angularsen/UnitsNet/blob/master/Common/UnitDefinitions/TemperatureDelta.json

Reply:
See the reply above about the singular/plural naming. However I agree adding Degree is a good idea. Consistency is always good!